sources/packages/backend/libs/services/src/students-assessments/assessment-sequential-processing.service.ts

```diff
@@ -542,11 +531,10 @@ export class AssessmentSequentialProcessingService {
       .addSelect("SUM(sfasApplication.sbsdAward)", "SBSD")
       .addSelect("SUM(sfasApplication.csgdAward)", "CSGD")
       .addSelect("SUM(sfasApplication.bcagAward)", "BCAG")
-      .innerJoin("sfasApplication.individual", "sfasStudent")
+      .innerJoin("sfasApplication.individual", "sfasIndividual")
+      .innerJoin("sfasIndividual.student", "student")
       .where("sfasApplication.applicationCancelDate IS NULL")
-      .andWhere("lower(sfasStudent.lastName) = lower(:lastName)", { lastName })
-      .andWhere("sfasStudent.sin = :sin", { sin })
-      .andWhere("sfasStudent.birthDate = :birthDate", { birthDate })
+      .andWhere("student.id = :studentId", { studentId })
```

Comment:
In the same way as the other, this one can also do the where condition on the student without the need to join with the student, right?